### ISSUE TITLE
Broken support for SQL files

### ISSUE THREAD

**Issue by aschempp** [contao/core-bundle#324]
The changes in https://github.com/contao/core-bundle/commit/ecc92b47e5cb356a982104be1e7aa6d11e9da669 break support for SQL files. If there are multiple files, the definition for `TABLE_OPTIONS` will become an array (due to https://github.com/contao/core-bundle/commit/ecc92b47e5cb356a982104be1e7aa6d11e9da669#diff-306aaced706707a680a1fa11ed948e81R447) which is invalid.

**Comment by leofeyer** [comment]
I actually cannot reproduce this. I have added two extensions with `database.sql` files (namely "dlh_googlemaps" and "conditionalselectmenu") and the SQL queries have been merged correctly.

**Comment by aschempp** [comment]
Both defining the same table? In my case the database.sql was extending tl_user (which was predefined by a DCA file).

**Comment by leofeyer** [comment]
Which extension did you use?

**Comment by aschempp** [comment]
easy_themes

**Comment by leofeyer** [comment]
How did you get this to work at all?

```
Warning: in_array() expects parameter 2 to be array, null given in ~/Sites/contao/standard-edition/vendor/contao/core-bundle/src/Resources/contao/library/Contao/System.php on line 172

Fatal error: Class 'EasyThemes' not found in ~/Sites/contao/standard-edition/vendor/contao/core-bundle/src/Resources/contao/library/Contao/System.php on line 172
```

**Comment by aschempp** [comment]
The latest develop branch works. But I have also changed the SQL-file to DCA, so you would need to test with the second last commit of develop.

**Comment by leofeyer** [comment]
Fixed in 16c62d554619b9c761f453eea1e7bae2ea5b5fbf.
